geode-core/src/main/java/org/apache/geode/internal/cache/xmlcache/CacheCreation.java

```diff
@@ -1115,6 +1116,11 @@ public void unlockDiskStore(String diskStoreName) {
 
   }
 
+  @Override
+  public boolean hasMemberOlderThan(KnownVersion version) {
+    return false;
```

Comment:
I wonder if, for this method, it would make sense to follow the approach used elsewhere in this class where we throw an UnsupportedOperationException rather than returning an effectively meaningless value. I think that there's no scenario in which it would make sense to call this method on a `CacheCreation`.